great_expectations/core/batch.py

```diff
@@ -261,7 +271,18 @@ def to_json_dict(self) -> dict:
         make this refactoring infeasible at the present time.
         """
         dict_obj: dict = self.to_dict()
+        batch_data = dict_obj.get("runtime_parameters", {}).get("batch_data")
+
+        if isinstance(batch_data, pd.DataFrame):
+            dict_obj["runtime_parameters"]["batch_data"] = "PandasDataFrame"
+
+        if pyspark and isinstance(batch_data, pyspark.sql.DataFrame):
```

Comment:
It would be helpful to have a note that pyspark may not have been imported, and we're checking that here

Reply:
Sure thing.